docs/releasenotes/RELEASE-NOTES-1.0.md

```
@@ -0,0 +1,155 @@
+# Semantic MediaWiki 1.0
+
+Released on December 31, 2007.
+
+## Changes in SMW1.0 as compared to SMW0.7
+
+* Simplified semantic annotations: just one kind of annotation ("Property").
+* Significant speedup (both server and network load substantially reduced,
+  faster RDF export, more efficient query result formatting).
+* Prettier and easier to understand interfaces:
+  ** New tooltips for warnings and additional information.
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)